### PR TITLE
Added generation and seed to Postgres

### PR DESCRIPTION
**Pull request by kazshige**
### Features:
-seeds scripts with helper.js
-integrated sequelize models, migrations, and seeders for Postgres

### Questions:
Do you think I made a good decision for not creating CSV or JSON? If not, can you share your thought?
Do you see any problems with my sequelize configurations?
Do you see any problems with my helper.js file?

Thank you for your help!

### CODE REVIEW

File: server/migrations/20190419001355-create-album.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('Albums');
+  }
+};
```
Comment (sydneycasey) [inline server/migrations/20190419001355-create-album.js:39]: This looks great! I've been trying to figure out my schema / seed scripts for postgres. I'm curious about what `up` and `down` are doing in `module.exports`? I haven't seen that before!

File: helper.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  var output = []
+  for (var i = 1; i <= 10; i++) {
+    writeAlbumData(start, stop, output);
+    start += one_million;
+    stop += one_million;
+  }
+  return output
+}
+
+module.exports = generateData
```
Comment (sydneycasey) [inline helper.js:44]: There are few missing semicolons in this file, otherwise looks great

**Review by sydneycasey — COMMENTED** [review]
This is looking great! Awesome use of promises and generally clean code. I left a couple of other comments in here. 👍

File: server/migrations/20190419001355-create-album.js (hunk 1 above)
Comment (kazshige) [reply server/migrations/20190419001355-create-album.js:39]: the "up" method is the specified method that will be run by sequelize to seed the database while the "down" is the method that will be run by sequelize to drop the seeded table in the database.

File: helper.js (hunk 2 above)
Comment (kazshige) [reply helper.js:44]: Thanks, great catch! I will fix them.

**Review by brockross — APPROVED** [review]
I like the way you set this up! I think your decision not to use JSON or CSV is fine--I've actually refactored to do the same. Your creation of models and interfaces is extremely clean and professional.

**Comment by kazshige** [comment]
Thanks for your review comments, Brock! Now I'm pretty confident in my approach.

**Pull request merged**